### PR TITLE
KIALI-1413 Support inaccessible components

### PR DESCRIPTION
**Pull request by mwringe**
Part of the change for KIALI-616 (see https://github.com/kiali/kiali/pull/448)

If we have nodes that are inaccessible, we shouldn't be fetching health information about them (since we don't have access to the k8s objects that are used in the health calculations).

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#36](https://paste.fedoraproject.org/paste/STupBfxO8Huq46At9~QXAA)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1232]](https://paste.fedoraproject.org/paste/hZ9J6l7tZ2XhUD1R5WszRg)
  * :heavy_check_mark: Kiali-on-Openshift [#[577]](https://paste.fedoraproject.org/paste/-atoFTLIp6YNZcbP-NP6-Q)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1008]](https://paste.fedoraproject.org/paste/vr~jgkMQbbsJMWzoN-kwKQ)
  * :red_circle: run-kiali-e2e-tests [#[57]](https://paste.fedoraproject.org/paste/GrtY8beKtOHgBG4hmY2wMg)


KIALI_NAME=kialiuipr611-istio-system

**Comment by mwringe** [comment]
The latest commit adds an icon when dealing with inaccessible nodes (and also disables double clicking on the node since we don't want to move the user to a view we know they can't access).

The current icon if the node exists in another namespace:

screenshot from 2018-09-13 18-06-19

The proposed icon in this pr for when the namespace is external and the current user doesn't have access to it (using the patternfly locked icon)

screenshot from 2018-09-13 18-06-22

@cshinn any comments on using the locked icon here? Its to represent its in another namespace and that the current user doesn't have access to it.

**Comment by mwringe** [comment]
Adding in the 'Do Not Merge' label just because we need the Kiali one to be merged first: https://github.com/kiali/kiali/pull/448

**Comment by jshaughn** [comment]
@mwringe The lock icon is already used for mutual_tls on the edges (I think).  So maybe this is a little confusing, I don't know.

**Comment by jmazzitelli** [comment]
> @mwringe The lock icon is already used for mutual_tls on the edges (I think). So maybe this is a little confusing, I don't know.

Yeah, that's the same icon. That may confuse people - they may think, "hmm, traffic to this namespace is secure via TLS" since that's the same icon used to on edge labels to represent mTLS secured traffic

**Comment by cshinn** [comment]
@mwringe there's also `pficon-private` which conveys a similar meaning with a different appearance. It's a bit closer to the topology icon and might be different enough to avoid confusion with secured edges

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#61](https://paste.fedoraproject.org/paste/UYbbkozq7-ZutLqBoGdXdw)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1283]](https://paste.fedoraproject.org/paste/E5xsfHCKsSplnFSZsCDZmQ)
  * :heavy_check_mark: Istio-Upstream [#[168]](https://paste.fedoraproject.org/paste/mxI7~VGGfriz2cTVXy-G6Q)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1084, 1085]](https://paste.fedoraproject.org/paste/b5A4tEeauPJuN4fgq-omrA)
  * :heavy_check_mark: install-bookinfo [#[633]](https://paste.fedoraproject.org/paste/goBv4d-bJXJSRrUR~lIYDw)
  * :heavy_check_mark: Kiali-on-Openshift [#[633]](https://paste.fedoraproject.org/paste/TJg9SoW47nTZOpKRqzKtfw)
  * :heavy_check_mark: run-kiali-e2e-tests [#[100]](https://paste.fedoraproject.org/paste/JS9he5D8aqI9T5aUduLWUA)


KIALI_NAME=kialiuipr611-istio-system

**Comment by mwringe** [comment]
Updated to just use the pficon-private icon instead, but its a bit weird with the circle:

screenshot from 2018-09-17 16-50-38

Playing around with it a bit more and I think this could work: 
screenshot from 2018-09-17 17-32-30

The pr has been updated to use the pficon-private with the gray background. Not sure if its still too similar to the lock icon we use on the edges for mtls.

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#62](https://paste.fedoraproject.org/paste/HsX5Qhglr0Tf~RYYNskYrQ)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1284]](https://paste.fedoraproject.org/paste/o62gkR8i2JUlCKF-GM6a9A)
  * :heavy_check_mark: Istio-Upstream [#[169]](https://paste.fedoraproject.org/paste/ixpXWn1lcmPD~~fH5Q6EXQ)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1086, 1087]](https://paste.fedoraproject.org/paste/4CZ4yiIqPRJJIpmQdCKVqw)
  * :heavy_check_mark: install-bookinfo [#[634]](https://paste.fedoraproject.org/paste/XJlA9voNmYAbMOf514vMDw)
  * :heavy_check_mark: Kiali-on-Openshift [#[634]](https://paste.fedoraproject.org/paste/8j2MQKWClJ5S4MEkn5HpCA)
  * :heavy_check_mark: run-kiali-e2e-tests [#[101]](https://paste.fedoraproject.org/paste/ICn0-73d8~H-gNF3cXLl6A)


KIALI_NAME=kialiuipr611-istio-system

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@kiali/kiali-ui",
   "version": "0.8.0",
+  "proxy": "https://kiali-istio-system.192.168.122.2.nip.io",
```
Comment (jshaughn) [inline package.json]: You don't want this!

Comment (mwringe) [reply package.json]: lol, whoops, I can't believe I didn't catch that before commit.

**Comment by mwringe** [comment]
The package.json file has been removed.

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#69](https://paste.fedoraproject.org/paste/266omnZD5b9duLWfZ5jQNw)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1292]](https://paste.fedoraproject.org/paste/vm2hyxyBUzi0lTVWbF45BQ)
  * :heavy_check_mark: Istio-Upstream [#[176]](https://paste.fedoraproject.org/paste/Ss67skk3pJ--33wxgwx-Cg)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1104, 1103]](https://paste.fedoraproject.org/paste/4Qu0sHfURjL2DYN8PjOJew)
  * :heavy_check_mark: install-bookinfo [#[642]](https://paste.fedoraproject.org/paste/OAj-gXkhA8xo8SLbD8KaTQ)
  * :heavy_check_mark: Kiali-on-Openshift [#[645]](https://paste.fedoraproject.org/paste/wiz9ynugRLD770EVlMTDJw)
  * :heavy_check_mark: run-kiali-e2e-tests [#[108]](https://paste.fedoraproject.org/paste/fWZ3ZuRfZdK23rixvz9x-w)


KIALI_NAME=kialiuipr611-istio-system

**Pull request merged**